tools/package.json

```diff
@@ -13,12 +13,12 @@
     "build": "tsc",
     "watch": "tsc --watch",
     "clean": "rm -rf build cache",
-    "et": "bin/expotools.js",
+    "et": "node bin/expotools.js",
     "lint": "eslint ./src"
   },
   "bin": {
-    "et": "bin/expotools.js",
-    "expotools": "bin/expotools.js"
+    "et": "node bin/expotools.js",
+    "expotools": "node bin/expotools.js"
```

Comment:
"bin" values need to be filenames, not commands. That said, maybe we can remove all of the "bin" entries since I believe expotools is not installed as a dependency anywhere.